README.md

```diff
@@ -16,7 +16,7 @@
 Pode is a Cross-Platform framework for creating web servers to host [REST APIs](https://badgerati.github.io/Pode/Tutorials/Routes/Overview/), [Web Pages](https://badgerati.github.io/Pode/Tutorials/Routes/Examples/WebPages/), and [SMTP/TCP](https://badgerati.github.io/Pode/Servers/) Servers. Pode also allows you to render dynamic files using [`.pode`](https://badgerati.github.io/Pode/Tutorials/Views/Pode/) files, which are just embedded PowerShell, or other [Third-Party](https://badgerati.github.io/Pode/Tutorials/Views/ThirdParty/) template engines. Plus many more features, including [Azure Functions](https://badgerati.github.io/Pode/Hosting/AzureFunctions/) and [AWS Lambda](https://badgerati.github.io/Pode/Hosting/AwsLambda/) support!
 
 <p align="center">
-    <img src="https://github.com/Badgerati/Pode/blob/develop/images/example_code_2.png?raw=true" />
+    <img src="https://github.com/Badgerati/Pode/blob/develop/images/example_code_readme.svg?raw=true" />
```

Comment:
The image looks much better! 😄 

For the `<img>` tag, just so it's scaled appropriately with everything else, adding either `width="70%"` or `width="80%"` will help so it's not overly "large"

That should be good to merge then! :)

Reply:
@Badgerati I've added the `width` attribute in the latest commit. 😄  Thanks for checking it out!